perceval/runtime/rpc_handler.py

```diff
@@ -39,57 +41,94 @@
 
 
 class RPCHandler:
+    """Remote Call Procedure Handler
+
+    A class to call the API
+
+    """
+
     def __init__(self, name, url, token):
+        """Remote Call Procedure Handler
+
+        :param name: name of the plateform
+        :param url: api URL to call
+        :param token: token used for identification
+        """
         self.name = name
         self.url = url
         self.token = token
-        self.headers = {
-            'Authorization': f'Bearer {token}'
-        }
+        self.headers = {'Authorization': f'Bearer {token}'}
+        self.def_t = 10  # default timeout
 
-    def build_endpoint(self, endpoint):
-        return f"{self.url}{endpoint}"
+    def build_endpoint(self, endpoint, *args):
+        """build the default endpoint url
+
+        :param endpoint: first part of the endpoint
+        :return: the full endpoint url from the args
+        """
+        endpath = ''
+        if len(args) > 0:
+            endpath = f"/{'/'.join(str(x) for x in args)}"
```

Comment:
`endpath` always starts with a slash. When endpoint ends with a slash too, there's a `//` in the resulting address.
If you look at the `_ENDPOINT_<whatever>` constants, they all finish with a slash.
So I guess that's a problem.